pkg/controller/sidecarset/sidecarset_processor.go

```diff
@@ -127,13 +142,13 @@ func (p *Processor) UpdateSidecarSet(sidecarSet *appsv1alpha1.SidecarSet) (recon
 	}
 
 	// 7. upgrade pod sidecar
-	if err := p.updatePods(control, pods); err != nil {
+	if err := p.updatePods(control, pods, latestRevision); err != nil {
```

Comment:
sidecarSet.status already contains the latestRevison, so don't need to add additional parameters latestRevision

Reply:
Do you mean we should execute "get status" in this function?